sqlglot/expressions.py

```diff
@@ -2353,6 +2353,10 @@ class Posexplode(Func):
     pass
 
 
+class Position(Func):
```

Comment:
is this the same as StrPosition? can we reuse that?

Reply:
Yes and no. It is in different syntax. From the [docs](https://www.postgresql.org/docs/current/functions-string.html):
```
 strpos ( string text, substring text ) → integer

Returns first starting index of the specified substring within string, or zero if it's not present. (Same as position(substring in string), but note the reversed argument order.)

strpos('high', 'ig') → 2
```
I wonder as well if we can somehow tie them together. @tobymao, do you suggest updating the parser to produce `StrPosition` object after parsing `POSITION(foo in bar)` expression?

Reply:
yes that’s what i’m suggesting

Reply:
@tobymao, I think it's done.

Reply:
Should we turn the tables and instead make `StrPosition` generate calls to standard functions, as opposed to vendor-specific functions, by default?